Documentation/ceph-cluster-crd.md

```diff
@@ -5,8 +5,30 @@ indent: true
 ---
 
 # Ceph Cluster CRD
-Rook allows creation and customization of storage clusters through the custom resource definitions (CRDs). The following settings are
-available for a Ceph cluster.
+Rook allows creation and customization of storage clusters through the custom resource definitions (CRDs).
+
+## Sample
+To get you started, here is a simple example of a CRD to configure a Ceph cluster with all nodes and all devices. More examples are included [later in this doc](#samples).
+
+```yaml
+apiVersion: ceph.rook.io/v1beta1
+kind: Cluster
+metadata:
+  name: rook-ceph
+  namespace: rook-ceph
+spec:
+  dataDirHostPath: /var/lib/rook
+  serviceAccount: rook-ceph-cluster
+  network:
```

Comment:
it seems like we're wanting to give the quickest/simplest example that will use all devices and nodes.  We can probably exclude the network/hostNetwork and dashboard fields to make this even more compact, but still useful.